packages/eslint-config-nextjs-ts/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@infinumjs/eslint-config-nextjs-ts",
+  "version": "3.2.0",
+  "description": "Infinum's ESLint NextJS TypeScript rules",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/infinum/js-linters.git"
+  },
+  "author": "Infinum JavaScript Team <hello@infinum.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/infinum/js-linters/issues"
+  },
+  "homepage": "https://github.com/infinum/js-linters#readme",
+  "dependencies": {
+    "@infinumjs/eslint-config-react-ts": "^3.2.0"
+  },
+  "peerDependencies": {
+    "eslint": ">=7.23.0",
+    "typescript": ">=3.3.1"
+  },
+  "devDependencies": {
+    "eslint": ">=7.23.0",
```

Comment:
This should now be `"eslint": ">=8.14"`, after [the latest version update](https://github.com/infinum/js-linters/releases/tag/v3.3.1).

Reply:
Done
https://github.com/infinum/js-linters/pull/66/commits/07dd0da48bf9dfe3b0da020dd59eec8cbb8f9132